core/primitives/src/telemetry.rs

```diff
@@ -37,4 +37,5 @@ pub struct TelemetryInfo {
     pub agent: TelemetryAgentInfo,
     pub system: TelemetrySystemInfo,
     pub chain: TelemetryChainInfo,
+    pub blob: String,
```

Comment:
nit: I think `blob: String` does not help a reader gain much insight into what this field is and how it will be used.  We could consider renaming blob to something like extra_telemetry_info but that is probably not future compatible.  Should we add some documentation here to help readers?

Reply:
Fair point, renamed to `extra_info`.